system/iptables_raw.py

```diff
@@ -272,14 +276,27 @@ def _get_list_of_active_tables(self):
         else:
             return []
 
+    #If /etc/debian_version exist, this means this is a debian based OS (Ubuntu, Mint, etc...)
+    def is_debian(self):
```

Comment:
Could you please name this method `_is_debian`, since all other "private" methods start with an underscore.